pkg/controller/boilerplate.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
```

Comment:
This file looks like a duplicate of  pkg/boilerplate/controller/controller.go

Reply:
Nice, this file is pretty old.